File: Oddystitches/Day-5/plusOne.py

class Solution(object):
    def plusOne(self,digits):
        while digits[0]!=0:
            if 1<=len(digits)<=100:
                digit= [str(i) for i in digits]
                res= int("".join(digit))
                res=res+1
                s=[int(i) for i in str(res)]
                return s



solution=Solution()
lis=[3,5,8,9]
print(solution.plusOne(lis))

File: Ola/Array/Day 5/plusone.js
//Solution
let digits=[1,2,3,5,6,9]

var plusOne = function(digits) {
  for(let i = digits.length -1;i>=0;i--){
      digits[i]++;
      if( digits[i]<10){
          return digits;
      }
      else{
          digits[i]=0;
      }
  }
  digits.unshift(1)
  return digits;
};
console.log(plusOne(digits));